### PR TITLE
Added moar languages

### PR DESCRIPTION
**Pull request by dane-johnson**
## Description

Added languages from http://spectrum.ieee.org/computing/software/top-10-programming-languages/ supported by highlight.js

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Browsers

- [ ] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Mobile:
- [ ] Other:

### CODE REVIEW

File: src/util/highlight.js [hunk 1]
```diff
@@ -3,13 +3,47 @@
...
+import lua from 'highlight.js/lib/languages/lua';
+import matlab from 'highlight.js/lib/languages/matlab';
+import perl from 'highlight.js/lib/languages/perl';
+import php from 'highlight.js/lib/languages/php';
 import python from 'highlight.js/lib/languages/python';
+import r from 'highlight.js/lib/languages/r';
+import ruby from 'highlight.js/lib/languages/ruby';
+import scala from 'highlight.js/lib/languages/scala';
+import sql from 'highlight.js/lib/languages/sql';
+import x86asm from 'highlight.js/lib/languages/x86asm';
```
Comment (thebho) [inline src/util/highlight.js]: Nope

Comment (dane-johnson) [inline src/util/highlight.js]: We should support an assembly language, this is the most popular/readable. I stand by it.

**Comment by thebho** [comment]
we should make a decision about this today

**Comment by solkaz** [comment]
I support adding bash, Javascript, C#, php, ruby and R because it's more likely that students at LaunchCode would know those languages already than the others provided. They will probably not know what ASM is, because that's too low-level for beginners to understand, nor will they be learning about Lisp anytime soon.

**Comment by dane-johnson** [comment]
I stand by assembly. Low level languages are something that I can see being referenced in a CS course.

**Comment by thebho** [comment]
Fine, any last complaints with this list?

**Comment by solkaz** [comment]
These are students learning the fundamentals of programming using Python and Java, two high level languages. I don't think they'll be getting anywhere near the metal, as it should  be at this stage in their education

**Comment by solkaz** [comment]
Having said that, I'm willing to accept ASM along with the languages I listed once all other languages are removed.

**Pull request merged**